src/linter.js

```diff
@@ -629,4 +634,25 @@ export default class Linter {
       });
   }
 
+  _markBadwordUsage(filename, fileData) {
+    if (fileData) {
+      const badwordsRe = new RegExp(badwords.en.join('|'), 'gi');
```

Comment:
This is prepared to match other languages but not really implemented yet unless we have a use-case for it.